lizmap/modules/lizmap/classes/qgisVectorLayerDatasource.class.php

```diff
@@ -28,8 +28,8 @@ class qgisVectorLayerDatasource
         'srid' => 'srid=([0-9]+) ',
         'type' => 'type=([a-zA-Z]+) ',
         'checkPrimaryKeyUnicity' => "checkPrimaryKeyUnicity='([0-1]+)' ",
-        'table' => ' table="(.+)" (\([^ ]+\) )?sql=',
-        'geocol' => '\(([^ ]+)\) sql=',
+        'table' => ' table="(.+)"',
```

Comment:
```suggestion
        'table' => ' table="(.+)" (\([^ ]+\) )?sql=',
```
Why ave you changed this line ?

Reply:
because on my server, the datasource doesn't contains 'sql=' :
The variable look like this :
"dbname='test_dbname' host=127.0.0.1 port=5432 user='test_user' password='test_password' sslmode=disable key='id' srid=4326 type=Point checkPrimaryKeyUnicity='0' table=\"test_schema\".\"test_table\" (geog)";
Maybe because it is a Geography database and not Geometry one ?

Reply:
Which QGIS version ?

Reply:
3.14.16